stats/measure_int64.go

```diff
@@ -38,15 +37,7 @@ func (m *MeasureInt64) Unit() string {
 	return m.unit
 }
 
-func (m *MeasureInt64) addView(v *View) {
-	m.views[v] = true
-}
-
-func (m *MeasureInt64) removeView(v *View) {
-	delete(m.views, v)
-}
-
-func (m *MeasureInt64) viewsCount() int { return len(m.views) }
+var _ Measure = (*MeasureInt64)(nil)
```

Comment:
Can we also do it for MeasureFloat64?

Reply:
Roger that, I'll do that too and fix the tests, thanks!